client/src/main/java/org/apache/celeborn/client/ReviveManager.java

```diff
@@ -124,4 +125,8 @@ public void addRequest(ReviveRequest request) {
       logger.error("Exception when put into requests!", e);
     }
   }
+
+  public void close() {
+    ThreadUtils.shutdown(batchReviveRequestScheduler, Duration.apply("800ms"));
```

Comment:
I'm not sure it's a good timeout, but make sense to me

Reply:
@pan3793, all the shutdown of executor service use the same timeout. cc @RexXiong.